promitor-agent-scraper/templates/serviceaccount.yaml

```diff
@@ -1,5 +1,6 @@
 {{- if and .Values.rbac.create .Values.rbac.serviceAccount.create -}}
 apiVersion: v1
+automountServiceAccountToken: false
```

Comment:
Shouldn't this be read from the value instead?

Reply:
In this case no. You can actually set automountServiceAccountToken in both ServiceAccount resource and on pod level. Pod level has the ability to override whatever is set on sa level. NSA thinks, and I tend to agree, that setting this on sa level is wrong because it might lead to accidental or undesired mounting of token. So the suggestion is always disable it on sa level and explicitly mount it on pod level if needed.